pandas/core/frame.py

```diff
@@ -1335,7 +1335,8 @@ def from_records(cls, data, index=None, exclude=None, columns=None,
 
         return cls(mgr)
 
-    def to_records(self, index=True, convert_datetime64=None):
+    def to_records(self, index=True, convert_datetime64=None,
```

Comment:
so this is an extremely misleading parameter name. These are NOT bytes, rather they are fixed length strings. maybe ``strings_as_fixed_length``

Reply:
on second though call this ``stringlike_as_fixed_length``, to encompass string, unicode, bytes